x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/embeddable/embeddable.scss

```diff
@@ -0,0 +1,33 @@
+.canvasEmbeddable {
+  .embPanel {
+    border: none;
```

Comment:
Maybe more of a @ryankeairns concern, but I think we should keep the border _at least_ in edit mode.  It feels weird having the gear hanging out in the top disconnected from everything, but the border at least boxes it in and gives you some clue that all of that is tied in together.